brambling/static/brambling/css/styles.css

```diff
@@ -1,7 +1,7 @@
 @charset "UTF-8";
 @import url(../fonts/stryde.css);
 @import url(../fonts/stryde-bold.css);
-/*! normalize.css v3.0.2 | MIT License | git.io/normalize */
+/*! normalize.css v3.0.1 | MIT License | git.io/normalize */
```

Comment:
You've decreased the version of normalize.css. Is your Bootstrap not up to date?